connect/src/state/context.rs

```diff
@@ -116,6 +116,10 @@ impl ConnectState {
             ResetContext::Completely => {
                 self.context = None;
                 self.autoplay_context = None;
+
+                let player = self.player_mut();
+                player.context_uri.clear();
+                player.context_url.clear();
```

Comment:
Need to clear twice?

Reply:
Spotify sends a context `uri` and a `url`. And we clear both here. I suppose you didn't see the `i` and `l` difference. Happens all the time heh

> I had a bug in the middle of developing with it, where I used the wrong variable and set the `uri` twice instead and did by that override the previous set... that was a pain to figure out :D

Reply:
Ahh yes... wow.